pandas/tools/tests/test_pivot.py

```diff
@@ -936,6 +936,84 @@ def test_crosstab_no_overlap(self):
 
         tm.assert_frame_equal(actual, expected)
 
+    def test_margin_ignore_dropna_bug(self):
+        # GH 12577
```

Comment:
Trying to make sure I got this right -  

I current have two test methods respectively for this one and #12614. They're both `margins` and `dropna` issues for sure.

And I should merge these two methods into one, and name it `test_margin_dropna`, correct?

And I'll keep the issue number here?